### DISCUSSION TITLE
Sending Databricks Token to Workers

### DISCUSSION THREAD

**Discussion by samuel-gerstein** [Nixtla/neuralforecast#1052] [Q&A]
Hi,

I am trying to use MLFlow with Spark distributed training. For TorchDistributor to log metrics however, it needs my Databricks token. In TorchDistributor's run function you could just initialize HOST_TOKEN, however there is no functionality to pass this in with Nixtla.

Is there a way to work around this?

**Comment by jmoralez** [comment]
I think you can set it as a cluster env variable
